tests/install_testbed/destroytestbeds.py

```diff
@@ -64,6 +56,15 @@ def destroy_testbeds():
         print "Deleting sec group " + sec.name
         nova.security_groups.delete(sec)
 
+    # Delete keypair
+    keys = nova.keypairs.findall(name=settings.key_name)
+    for key in keys:
+        print "Deleting keypair " + key.name
+        nova.keypairs.delete(key)
+        filename = os.path.expanduser('~/.ssh/' + settings.key_name)
```

Comment:
Just two comments, how can you be sured that the key is in this directory and the name does not include .pem extension?